consensus/types/src/chain_spec.rs

```diff
@@ -1296,6 +1311,10 @@ pub struct Config {
     #[serde(with = "serde_utils::address_hex")]
     deposit_contract_address: Address,
 
+    #[serde(default = "default_gas_limit_adjustment_factor")]
+    #[serde(with = "serde_utils::quoted_u64")]
+    gas_limit_adjustment_factor: u64,
```

Comment:
I wonder if we should leave this out of `Config` so that users _can't_ set it? It's not defined as part of the upstream `config.yaml` that the EF uses, so might make sense not to add it here as a non-standard config.

Then again, for most configs it just won't be present so this doesn't pose an issue.